js/index.js

```diff
@@ -141,36 +168,51 @@ function load() {
 }
 
 function populateSettings() {
-    for (let key in savedata) {
-        if (!(key in keySettingMapInverse)) continue;
-        let value = savedata[key];
-        let id = keySettingMapInverse[key];
-        
-        const input = document.querySelector("#" + id);
-        if (input.type === "checkbox") {
-            if (value === true || value === false) {
-                input.checked = value;
+    // Populate selection checkboxes
+    for (const [id, prop] of Object.entries(keySettingMap)) {
+        const input = document.getElementById(id);
+        if (input) {
+            if (typeof savedata[prop] === 'boolean') {
+                input.checked = savedata[prop];
+            } else if (typeof savedata[prop] === 'string' || typeof savedata[prop] === 'number') {
+                input.value = savedata[prop];
             }
         }
-        else if (input.type === "number") {
-            if (!value && isKeyNullable(id)) {
```

Comment:
While I approve of the getting rid of keySettingMapInverse since it serves no purpose, I believe these checks served some purpose in case weird profiles get shared/or are out of date. I'd need to test to figure that out.